CustomItems/Items/C4Charge.cs

```diff
@@ -0,0 +1,227 @@
+                },
+
+                new DynamicSpawnPoint
+                {
+                    Chance = 50,
+                    Location = SpawnLocation.Inside049Armory,
+                },
+
+                new DynamicSpawnPoint
+                {
+                    Chance = 100,
+                    Location = SpawnLocation.InsideSurfaceNuke,
+                },
+            },
+        };
+
+        /// <inheritdoc/>
+        public override string Description { get; set; } = "Explosive charge that can be remotly detonated.";
+
+
+        /// <summary>
+        /// Gets or sets a value indicating whether C4 charge should stick to walls / ceiling.
+        /// </summary>
+        [Description("Should C4 charge stick to walls / ceiling.")]
+        public bool IsSticky { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating throwing force muliplier.
+        /// </summary>
+        [Description("Defines how strongly C4 will be thrown")]
+        public float ThrowMultiplier { get; set; } = 40f;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether C4 charge require a specific item to be detonated.
+        /// </summary>
+        [Description("Should C4 require a specific item to be detonated.")]
+        public bool RequireDetonator { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the Detonator Item that will be used to detonate C4 Charges.
+        /// </summary>
+        [Description("The Detonator Item that will be used to detonate C4 Charges")]
+        public ItemType DetonatorItem { get; set; } = ItemType.Radio;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether C4 charges be defused and dropped as pickable item when the player who placed them dies.
+        /// </summary>
+        [Description("Should C4 charges be defused and dropped as pickable item when the player who placed them dies.")]
+        public bool DropChargesOnDeath { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether maximum distance between C4 Charge and player to detonate.
+        /// </summary>
+        [Description("Maximum distance between C4 Charge and player to detonate.")]
+        public float MaxDistance { get; set; } = 100f;
+
+        /// <summary>
+        /// Gets or sets time after which the C4 charge will automatically detonate.
+        /// </summary>
+        [Description("Time after which the C4 charge will automatically detonate.")]
+        public override float FuseTime { get; set; } = 9999f;
+
+        /// <inheritdoc/>
+        [YamlIgnore]
+        public override bool ExplodeOnCollision { get; set; } = false;
+
+
+        /// <inheritdoc/>
+        protected override void SubscribeEvents()
+        {
+            Instance = this;
+
+            ServerEvent.WaitingForPlayers += OnWaitingForPlayers;
+
+            PlayerEvent.ThrowingGrenade += OnThrowing;
```

Comment:
You're registering to an event that's already registered in the base method.

Reply:
Fixed